.circleci/config.yml

```diff
@@ -2,8 +2,12 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@3.2.2
   node: circleci/node@5.0.2
+
 jobs:
-  build:
+  test:
```

Comment:
Changing the name to test so that the intent is clearer also goes better with the concept of "flaky"